recipes/markdownlint-cli2/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "markdownlint-cli2" %}
+{% set version = "0.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://registry.npmjs.org/markdownlint-cli2/-/markdownlint-cli2-{{ version }}.tgz
+  sha256: db7d09de0f934fb8146b5a2a01a819faf361aff36099de3030825d75ae1c4178
+
+build:
+  noarch: generic
+  number: 0
+  script: |
+    tgz=$(npm pack)
+    npm install -g $tgz
+
+requirements:
+  build:
+    - nodejs
+
+test:
+  commands:
+    - npm list -g markdownlint-cli2
```

Comment:
Shouldn't this expose a CLI you can query using `--help`?

Reply:
```suggestion
    - markdownlint-cli2 --help
```